lab-11-JR/server.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
```

Comment:
The server.js file needs to be simplified a lot. All of these routes have pretty much already been defined in the note-routes.js file, so you can just instantiate an express.Router() instance, and pass that router into your notes router via dep injection.